### PR TITLE
Added Lower Bound for DMI of Dry Cows in VS Calculations

### PR DESCRIPTION
**Pull request by tomhuhh**
As discussed with Kristan, a lower bound of 7.1 kg is set for dry matter intake of dry cows when calculating volatile solids based on data by Appuhamy et al. (2018).

### CODE REVIEW

File: RUFAS/routines/animal/manure/dry_cow_manure_excretion.py [hunk 1]
```diff
@@ -92,7 +92,15 @@ def manure_calculations(ration_formulation,
...
-    organic_matter_intake = dry_matter_intake - ash_diet_content
+    # Dry cow dry matter intake lower bound from Appuhamy 2018
+    Dry_DMI_Lower_Bound = 7.1  # kg
+    if dry_matter_intake >= Dry_DMI_Lower_Bound:
+        organic_matter_intake = dry_matter_intake * \
+            (100 - ASH_concentration) / 100
+    else:
+        dry_matter_intake = Dry_DMI_Lower_Bound
+        organic_matter_intake = dry_matter_intake * \
+            (100 - ASH_concentration) / 100
```
Comment (ltp23) [inline RUFAS/routines/animal/manure/dry_cow_manure_excretion.py]: To shorten it a little bit,

```
Dry_DMI_Lower_Bound = 7.1 # kg, we can move this to a constants file later
dry_matter_intake = max(dry_matter_intake, Dry_DMI_Lower_Bound)
organic_matter_intake = dry_matter_intake * \
            (100 - ASH_concentration) / 100
```

**Review by ltp23 — APPROVED** [review]
It looks good to me. Thank you, Haowen!

File: RUFAS/routines/animal/manure/dry_cow_manure_excretion.py (hunk 1 above)
Comment (JoeWaddell) [reply RUFAS/routines/animal/manure/dry_cow_manure_excretion.py]: Great comment, as this is something that I implemented (thanks to a suggestion by Yijing) in another PR! 

Also, as you noted, a lot of these magic values will move to the `animal_module_constants.py` files once PR #366 is merged in! I've added a few TODOs in my PRs to that effect.

**Review by JoeWaddell — APPROVED** [review]
Looks good to me: and Loi's suggestion makes the code more succinct and I think still maintains readability.

**Pull request merged**